### ISSUE TITLE
Location not enabling automatically

### ISSUE THREAD

**Issue by Abhijithsp** [Baseflow/flutter-geolocator#479]
## 🐛 Bug Report

Step 1: when i press a button it will open the default popup for enable the location service,as i inspected it doesn't enable the location.

Step 2: If i enable the location manually i am getting the current location,if i do the first step it doesn't work

  ```
Future<void> _getCurrentLocation() async {
    final Geolocator geolocator = Geolocator()..forceAndroidLocationManager;
      LocationOptions(accuracy: LocationAccuracy.high, distanceFilter: 10);
try {
        final position = await geolocator.getCurrentPosition(
          desiredAccuracy: LocationAccuracy.best,
        );
        setState(() {
          _currentPosition = position;
          print(_currentPosition);
        });
             SharedPrefrence().setLatitude(_currentPosition.latitude.toString());
              SharedPrefrence().setLongitude(_currentPosition.longitude.toString());
     );
      } catch (e) {
        final snackBar = SnackBar(content: Text(e.toString()));
        _scaffoldKey.currentState.showSnackBar(snackBar);
      }
 
  }

```

**Version:** geolocator ^5.3.2+2
flutter 1.17.5
**Platform:** 
- [ ] :robot: Android

**Comment by Abhijithsp** [comment]
No problem yet,still testing with production app @mvanbeusekom

**Comment by mvanbeusekom** [comment]
That is awesome @Abhijithsp thanks a lot! 

Just a quick update, we did have some feedback of small bugs that have already been fixed and we are now on [6.0.0-rc.4](https://pub.dev/packages/geolocator/versions/6.0.0-rc.4)

**Comment by Abhijithsp** [comment]
Will do @mvanbeusekom
